sympy/solvers/solveset.py

```diff
@@ -2171,7 +2171,7 @@ def solveset(f, symbol=None, domain=S.Complexes):
         raise ValueError("%s is not a valid SymPy expression" % f)
 
     if not isinstance(symbol, (Expr, Relational)) and  symbol is not None:
-        raise ValueError("%s is not a valid SymPy symbol" % symbol)
+        raise ValueError("%s is not a valid SymPy symbol" % (symbol,))
```

Comment:
Isn't this ok?
```python
>>> '%s' % 1
'1'
```

Reply:
The problem is with `% obj` where `obj` is arbitrary because `%` behaves differently if the object is a tuple:
```python
In [1]: '%s' % (1, 2)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-1566f083d307> in <module>
----> 1 '%s' % (1, 2)

TypeError: not all arguments converted during string formatting
```